frontend/src/scenes/App.tsx

```diff
@@ -72,16 +110,17 @@ export function App(): JSX.Element | null {
     }, [scene, user])
 
     if ((userLoading && !user) || (preflightLoading && !preflight)) {
-        return <SceneLoading />
+        return spinner
```

Comment:
I think this could be abstracted to `appLogic` too. We also can't display the AppScene if preflight hasn't loaded.

Reply:
makes sense and moved there